client/js/util.js

```diff
@@ -9,7 +9,7 @@ const KEYS = {
 };
 
 const nodeList = (nl) => {
-  return [...nl];
+  return Array.prototype.slice.call(nl || []);
```

Comment:
> Could we not use babel-polyfill?

I'm not familiar with it – I had thought that the es2015 preset we're using was the magic required to make our code compatible with es2015 browsers (obviously not I guess).

We should definitely use something that makes this happen if we can.